Cals video series part two (#661)
ember-learn/ember-blog: source/2020-05-29-the-ember-times-issue-150.md

---

## ["My Experience with Ember.js" Part 2 🎬](https://www.youtube.com/watch?v=R2JdP4lb5Xw)

> [Cal Woolgar (@callam_woolgar)](https://twitter.com/callam_woolgar) has a second short video out with his new series, [Creating an Ember Application!](https://www.youtube.com/watch?v=R2JdP4lb5Xw) He gets into a little more detail in this video and demonstrates how easy it is to install and get started with Ember.js and it's cli.

Reviewer: (Rhetoric questions to myself) Is "Creating an Ember Application" the name of the video or of the new series? "More detail" about what?

I thought the quoted parts were a bit ambiguous. I think these sentences can be more clear if we write,
```suggestion
[Cal Woolgar (@callam_woolgar)](https://twitter.com/callam_woolgar) has a second short video out, called [Creating an Ember Application](https://www.youtube.com/watch?v=R2JdP4lb5Xw)! He demonstrates how the Ember CLI makes it easy to install and get started with Ember.js.
```

```bash
npm install -g ember-cli
```

> This video touches on an [Ember app's anatomy](https://guides.emberjs.com/release/getting-started/anatomy-of-an-ember-app/) and how to create a basic [Ember.js](https://emberjs.com/) app using the ember cli for a fully working application with **one line of code!** 🎉

Reviewer: I think the second part of this sentence, starting with "how to create," is a bit too long.

It seemed to me like you wanted to emphasize an Ember app's well-defined project structure and how a reader can start with a single-line command.

What do you think about something like:
```suggestion
To create an Ember app with default [project structure](https://guides.emberjs.com/release/getting-started/anatomy-of-an-ember-app/):
```

```bash
ember new "app-name"
```

> And with that one line of code, Ember's cli installs all your dependencies for your new app. And you can also create [routes](https://guides.emberjs.com/release/routing/defining-your-routes/) and [templates](https://guides.emberjs.com/release/components/) with the cli. 

Reviewer: Let's capitalize the word CLI since it's an acronym. In speaking, we tend to start a sentence with an and, to continue a thought. Often (or probably), I think, the conjunction's not necessary.
```suggestion
With that one line of code, Ember's CLI installs all your dependencies for your new app. You can also create [routes](https://guides.emberjs.com/release/routing/defining-your-routes/) and [templates](https://guides.emberjs.com/release/components/) with the CLI. 
```

```bash
ember generate template "template-name"
ember generate route "route-name"
```

> There is a lot more info in his video  on these topics, so check out his series.  These are great videos for someone learning, look forward to the next video from cal! 😃

Reviewer: ```suggestion
There is a lot more info in his video on these topics, so check out his series.  These are great videos for someone learning, look forward to the next video from Cal! 😃
```
